src/app/frontend/deploy/deployfromsettings_controller.js

```diff
@@ -183,6 +190,33 @@ export default class DeployFromSettingsController {
   }
 
   /**
+   * @returns {!angular.$q.Promise}
+   * @export
+   */
+  updateAvailableProtocols() {
```

Comment:
If you say it this way then sure, it looks like validator ;). I will change it to validator.. Unfortunately this has to wait till the next week.